MapboxDirections/Attributes/MBSpeedLimit.swift

```diff
@@ -0,0 +1,118 @@
+import Foundation
+
+/**
+ A localized limit for measuring speed limits.
+ */
+@objc(MBSpeedLimit)
+public class SpeedLimit: NSObject, NSSecureCoding {
+    
+    /**
+     A unitless measure of speed which is dependent on the `MaximumSpeedLimit.speedUnits`.
+     
+     If a speed is not known, it will default to `Int.NSNotFound`.
+     */
+    @objc public var speed: Int = NSNotFound
+    
+    
+    /**
+     Units for `MaximumSpeedLimit.speed`.
+     */
+    @objc public var speedUnits: SpeedUnit = .none
```

Comment:
Rename this property to `unit`. “Speed” is already part of the containing type’s name.